app.js

```diff
@@ -341,17 +341,16 @@ const topicExists = function topicExists(id) {
   return docManager.fileExists(fileName, id);
 };
 
-// TODO change to a DELTE /topic/:id
-app.delete('/file', (req, res) => {
+app.delete('/topic/:id', (req, res) => {
```

Comment:
Can a assigned student delete the content, with all it's history, or should this only be done by Supervisors?
I only Supervisors, we should check permission first.